### DISCUSSION TITLE
RAG - get answers that contains exact copy of text from document

### DISCUSSION THREAD

**Discussion by Oddivh** [open-webui/open-webui#2540] [Q&A]
I'm making a RAG based on one single PDF document. The document contains requirements formatted as numbered lists in addition to generic text. It would be great if the answer to a user's input could include a list of relevant requirements in addition to a standard answer.
I guess this is not straightforward forward, only uploading the pdf. But it could it be an idea to extract these requirements into another document (and format - CSV?), and the RAG uses both these two documents. The pdf to answer generically, and the additional document to pick the relevant requirements?

**Comment by JuanjoJG-WATA** [comment]
I think you could do that in the most recent update, as now you can create models that straight get info from specific documents.
